docs/advanced/node-local-storage.md

```diff
@@ -0,0 +1,356 @@
+# Using node local storage on kubernetes
+
+This guide shows different ways to use node local on Lokomotive, to achieve near
+bare-metal performance inside Kubernetes pods.
+
+Before you continue, make sure to read generalities about these concepts:
+ * Volumes: https://kubernetes.io/docs/concepts/storage/volumes/
+ * Persistent Volumes and PVC: https://kubernetes.io/docs/concepts/storage/persistent-volumes/
+ * StorageClasses: https://kubernetes.io/docs/concepts/storage/storage-classes/
+
+More detail about specific volumes used and how to use them on lokomotive will
```

Comment:
This sentence doesn't make sense to me.

>More details about specific volumes used

used?

```suggestion
More details about specific volumes used and how to use them on Lokomotive will
```

Reply:
What I wanted to say is that read only the generalities and not everything about every volume possible (like Local, NFS, etc.) and more details about the types of volume used in the guide will follow.

Would something like:

```
More detail about specific types of volumes used and how to use them on Lokomotive will
follow, so no need to look into volume specifics on the first read.
```
make more sense to you?